curriculum/challenges/english/blocks/lab-pricing-plans-layout/68e45a2b4ab9b2f48b1cc6df.md

```diff
@@ -0,0 +1,312 @@
+
+**Note:** Be sure to link your stylesheet in your HTML and apply your CSS.
+
+# --hints--
+
+You should have a `h1` element with the text `Pricing Plans`.
+
+```js
+const h1 = document.querySelector('h1');
+assert.strictEqual(h1.textContent, 'Pricing Plans', 'The h1 has the text Pricing Plans');
+```
+
+You should have three `div` elements with the class `pricing-card`.
+
+```js
+const div = document.querySelectorAll('.pricing-card');
+assert.lengthOf(div, 3, 'There are 3 div elements with the class pricing-card');
+```
+
+You should have a `div` element with the class `pricing-card` and `basic-plan`.
+
+```js
+const div = document.querySelector('.pricing-card.basic-plan');
+assert.strictEqual(div.tagName, 'DIV');
+assert.isTrue(div.classList.contains('basic-plan'), 'The div contains the basic-plan class');
+assert.isTrue(div.classList.contains('pricing-card'), 'The div contains the pricing-card class');
+```
+
+You should have a `div` element with the class `pricing-card` and `premium-plan`.
+
+```js
+const div = document.querySelector('.pricing-card.premium-plan');
+assert.strictEqual(div.tagName, 'DIV');
+assert.isTrue(div.classList.contains('premium-plan'), 'The div contains the premium-plan class');
+assert.isTrue(div.classList.contains('pricing-card'), 'The div contains the pricing-card class');
+```
+
+You should have a `div` element with the class `pricing-card` and `pro-plan`.
+
+```js
+const div = document.querySelector('.pricing-card.pro-plan');
+assert.strictEqual(div.tagName, 'DIV');
+assert.isTrue(div.classList.contains('pro-plan'), 'The div contains the pro-plan class');
+assert.isTrue(div.classList.contains('pricing-card'), 'The div contains the pricing-card class');
+```
+
+Each of your `.pricing-card` elements should use Flexbox to stack its content in a column and justify the space between the children using `space-between`. Each element should also set the `flex` property to `0 0 200px` to give it a consistent width and prevent it from growing or shrinking in the layout.
+
+```js
+const helper = new __helpers.CSSHelp(document);
+assert.strictEqual(helper.getStyle('.pricing-card').getPropVal('flex'), '0 0 200px');
+assert.strictEqual(helper.getStyle('.pricing-card').getPropVal('flex-direction'), 'column');
+assert.strictEqual(helper.getStyle('.pricing-card').getPropVal('display'), 'flex');
+assert.strictEqual(helper.getStyle('.pricing-card').getPropVal('justify-content'), 'space-between');
+```
+
+The `basic-plan` element should be the first element in the layout.
+
+```js
+const helper = new __helpers.CSSHelp(document);
+assert.strictEqual(helper.getStyle('.basic-plan').getPropVal('order'), '0');
+```
+
+The `pro-plan` element should be the second element in the layout and have a `flex-grow` property.
+
+```js
+const helper = new __helpers.CSSHelp(document);
+assert.strictEqual(helper.getStyle('.pro-plan').getPropVal('order'), '1');
+assert.strictEqual(helper.getStyle('.pro-plan').getPropVal('flex-grow'), '2');
+```
```

Comment:
this should be broken up into two sepeate tests and hints

- one test and hint for the order
- one test and hint for the flex grow

also the hint right now doesn't mention it should be set to 2. the directions say that but the hint doesn't